model/src/main/proto/oppia_logger.proto

```diff
@@ -118,7 +118,45 @@ message EventLog {
   }
 }
 
-// Structure of a container that contains event logs.
-message EventLogs{
+// Structure for an exception log.
+message ExceptionLog {
+  // The timestamp of the exception.
+  int64 timestamp = 1;
+  // The stack trace of the exception.
+  int64 stacktrace = 2;
```

Comment:
I dont think this would be an int64 type, have you tried serializing the exception? it would most likely be either a string or a byte array (more likely the latter)

Reply:
Ya, I was a bit confused about it too. I have added a new message for that now. The stack trace element consists of 4 things-- declaring class, method name, file name and line number which make up the whole stack trace. So I have just used that thing here. I think it will work. PTAL :)